python/ray/tests/test_iter.py

```diff
@@ -158,6 +158,30 @@ def test_for_each(ray_start_regular_shared):
     assert list(it.gather_sync()) == [0, 4, 2, 6]
 
 
+def test_for_each_concur(ray_start_regular_shared):
+    from time import sleep, perf_counter
```

Comment:
Is there a reason for importing inside of the function? If not, prefer to import at top of file for standardization